### PR TITLE
Add ServiceBus inteface definition

### PR DESCRIPTION
**Pull request by elprans**

### CODE REVIEW

**Comment by pragnagopa** [comment]
Can you add the test code as well? I need to look at how this is being used

**Comment by elprans** [comment]
Test code is in the worker repo.  These are abstract definitions.

File: azure/functions/_servicebus.py [hunk 1]
```diff
@@ -0,0 +1,90 @@
+
+import abc
+import datetime
+import typing
+
+
+class ServiceBusMessage(abc.ABC):
```
Comment (pragnagopa) [inline azure/functions/_servicebus.py]: Azure Functions supports [ServiceBusQueues](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-queues-topics-subscriptions#queues) and [ServiceBusTopics](https://docs.microsoft.com/en-us/azure/service-bus-messaging/service-bus-queues-topics-subscriptions#topics-and-subscriptions). Add ServiceBusTopic

Comment (pragnagopa) [inline azure/functions/_servicebus.py]: Rename to ServiceBusQueueMessage

Comment (elprans) [reply azure/functions/_servicebus.py]: Do topics and queues actually receive different data?  The binding documentation mentions only that the parameter type may receive instances of [Message](https://docs.microsoft.com/dotnet/api/microsoft.azure.servicebus.message)

Comment (pragnagopa) [reply azure/functions/_servicebus.py]: @asavaritayal  - Can you look into this and confirm if we need different types for queue message and topic?

Comment (asavaritayal) [reply azure/functions/_servicebus.py]: I don't think we need different types. `function.json` should take care of the exact source.

**Pull request merged**